src/transformers/models/t5/modeling_tf_t5.py

```diff
@@ -1617,18 +1547,19 @@ def get_encoder(self):
 
     @add_start_docstrings_to_model_forward(T5_ENCODER_INPUTS_DOCSTRING)
     @replace_return_docstrings(output_type=TFBaseModelOutput, config_class=_CONFIG_FOR_DOC)
+    @unpack_inputs
     def call(
         self,
-        input_ids,
-        attention_mask=None,
-        head_mask=None,
-        inputs_embeds=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-        training=False,
+        input_ids: TFModelInputType,
```

Comment:
```suggestion
        input_ids: Optional[TFModelInputType] = None,
```

Reply:
This fix should make the test pass :) it is a missing default argument -- the PyTorch model has it, other classes in this file have it as well. 

To be fair, it is quite unlucky that our tests did not catch it before. Both `input_processing` and `unpack_inputs` rely on having a default for the main data input.